### DISCUSSION TITLE
Semantic Segmentation 기능 추가

### DISCUSSION THREAD

**Discussion by wwwwje2008** [Looking-4-Attention/AttentionMe#3] [Announcements]
main에 Semantic Segmentation 기능 업로드했습니다.
#1 에서 댓글로 나눠주셨던 기능들 하나씩 개발해주시면 될 것 같아요!

**Comment by Namhoon-Cho** [comment]
Zoom_in 기능 추가하신 것 확인했습니다. 저도 pointing 기능을 추가해보겠습니다. 지금까지 확대, 포인팅, 배경 밝기 변경 기능을 생각했죠 저희?

**Comment by iamdami** [comment]
<기능>
1.지정한 사람을 제외한 나머지(배경 포함)을 blur 처리
2.지정한 사람만 crop해서 저장
3.지정한 사람을 제외한 나머지(배경 포함)을 brighten 처리
4.지정한 사람을 제외한 나머지(배경 포함)을 darken 처리
4.지정한 사람을 화살표 pointing 하기
5.지정한 사람만 확대

<branch 종류>
1. main
- 이미지를 각 카테고리로 세그멘테이션하고 그 중에 원하는 사람의 넘버를 사용자가 고르면 그 사람에 해당하는 세그멘테이션 값과 그 세그멘테이션을 제외한 배경의 값 리턴

2. feat2-crop
- 지정한 사람(사용자가 고른 넘버의 사람)만 crop해서 저장

3. feat3-brighten
- 지정한 사람을 제외한 나머지(배경 포함)을 brighten 처리

4. feat4-darken
- 지정한 사람을 제외한 나머지(배경 포함)을 darken 처리

5. feat5-zoom-in
- 지정한 사람만 확대


6. feat6-pointing
- 지정한 사람을 화살표로 pointing 하기

**Reply by iamdami** [reply]
이런 식으로 진행하면 어떨까요?

**Reply by wwwwje2008** [reply]
좋아요!
그리고 brighten 이랑 darken은 하나로 합쳐도 괜찮을 것 같아요!

**Reply by hyun1227** [reply]
그럼 함수를 하나로 만들어 볼게요!

**Reply by wwwwje2008** [reply]
zoom-in은 지정한 사람을 가운데로 오게 자르는 기능으로 하고
enlargement는 배경은 그대로 두고 지정한 사람만 확대하는 기능으로 하면 좋을 것 같아용

**Comment by hyun1227** [comment]
저도 배경밝기를 조정하는 기능을 추가 했습니다

**Comment by iamdami** [comment]
crop 기능 추가했습니다
